src/carousel/carousel.scss

```diff
@@ -96,12 +96,16 @@
       width: $iui-icons-small;
       height: $iui-icons-small;
       border-radius: 50%;
-      transition: background-color $iui-speed-fast ease;
+      box-sizing: border-box;
+      border: 1px solid transparent;
+      transition: background-color $iui-speed-fast ease, box-shadow $iui-speed-fast ease;
       @media (prefers-reduced-motion: no-preference) {
-        transition: background-color $iui-speed-fast $iui-speed-fast ease, transform $iui-speed ease;
+        transition: background-color $iui-speed-fast $iui-speed-fast ease, transform $iui-speed ease,
+          box-shadow $iui-speed-fast ease, border-color $iui-speed-fast ease;
```

Comment:
These changes are going to conflict with #544 so we should decide now how we want to handle that

Reply:
If you are planning to come into the office tomorrow, let's figure it out in person.